cmd/observer/node_builder/observer_builder.go

```diff
@@ -863,6 +881,8 @@ func (builder *ObserverServiceBuilder) Initialize() error {
 
 	builder.enqueueConnectWithStakedAN()
 
+	builder.BuildConsensusFollower()
```

Comment:
this doesn't belong in the `Initialize` method. what issue did you run into?

Reply:
> this doesn't belong in the `Initialize` method. what issue did you run into?
Moved it here because the consensus follower initializes `Committee` and `Committee` is needed for initializing the RPC server(for SignerIndiciesDecoder).

Reply:
Moved back to build with other functions.